### DISCUSSION TITLE
More than 4 parallel testing is not working in keda selenium

### DISCUSSION THREAD

**Discussion by ajinkya15051994** [kedacore/keda#3749] [Ideas]
As part of an initial spike/POC I deployed Keda as part of eks core components and the keda selenium scalers in the selenium deployment directory to the infra cluster. We successfully see chrome nodes spun up (pods) when running tests. Running up to 4 parallel sessions works without issues but we see issues with the tests from 5 onwards. We will need to look into configurations for keda and selenium to enable us to run ~20 sessions for one test. How can I achieve this?

**Comment by JorTurFer** [comment]
Hey,
You should check in selenium side, KEDA doesn't manage anything in selenium, it just checks the metrics for scaling based on them
